client/client.go

```diff
@@ -24,8 +26,11 @@ import (
 )
 
 var (
-	myFqdn   = kingpin.Flag("fqdn", "FQDN to register with").Default(fqdn.Get()).String()
-	proxyURL = kingpin.Flag("proxy-url", "Push proxy to talk to.").Required().String()
+	myFqdn     = kingpin.Flag("fqdn", "FQDN to register with").Default(fqdn.Get()).String()
+	proxyURL   = kingpin.Flag("proxy-url", "Push proxy to talk to.").Required().String()
+	caCertFile = kingpin.Flag("cacert", "<file> CA certificate to verify peer against").String()
```

Comment:
The flag names could make it more obvious they're tls related

Reply:
@brian-brazil, `fqdn` and `proxy-url` were there already. I have added `cacert`,`cert` and `key` referring names and description from `curl` command.

Reply:
These names are a bit terse, and you'd have to read the code to release they were related to tls.

Reply:
Will prefixing flags with `tls_` will be sufficient?